spec/slack/web/faraday/response/raise_error_spec.rb

```diff
@@ -57,5 +57,15 @@
         )
       end
     end
+
+    context 'with non-JSON response body' do
```

Comment:
This should just be status 500 with a non-JSON body and we should get the expected error.